tests/h/subscribers_test.py

```diff
@@ -107,66 +99,63 @@ def event(self, pyramid_request):
 
 @pytest.mark.usefixtures("fetch_annotation")
 class TestSendReplyNotifications:
```

Comment:
I just rewrote these from scratch using normal mocking stuff. It ends up a plumbing test, but hey ho